### PR TITLE
Adjusting our ANML Model for Accuracy

### PR DESCRIPTION
**Pull request by mvacaporale**
This PR makes two minor but important changes:
1. We now use a batch_size=1 during meta-testing
2. We now use GroupNorm instead of BatchNorm2d in our ANML network

The motivation of 2 is the observation of the following line:
https://github.com/uvm-neurobotics-lab/ANML/blob/cabaaf7f2336496cd51065847c524a705408d562/model/learner.py#L188

Here, we see that ANML always run it's forward pass on each image one at a time. Thus, to replicate their network, we cannot compute statistics over a whole mini-batch. In this PR, GroupNorm ensures we do just that, while still normalizing each channel and transforming by gamma and beta parameters.

### CODE REVIEW

**Comment by mvacaporale** [comment]
I'm running meta-training and meta-testing now to verify the accuracy.

**Comment by mvacaporale** [comment]
Meta-Test Training Accuracies (ANML Repo with Vernon with GroupNorm and InstanceNorm2d)
image

**Comment by mvacaporale** [comment]
Meta-Test Testing Accuracies (ANML Repo with Vernon with GroupNorm and InstanceNorm2d)

image

**Comment by mvacaporale** [comment]
The Vernon model is finally performing up to par. I'll stick with InstanceNorm2d as it's simpler implementation-wise.

File: projects/meta_cl/experiments/anml_replicate.py [hunk 1]
```diff
@@ -111,7 +111,7 @@ def __call__(self, pil_image):
     num_meta_testing_runs=10,
 
     # Run through meta-test testing 5 images at a time. No training occurs here.
-    test_test_batch_size=5,
+    test_test_batch_size=1,
```
Comment (lucasosouza) [inline projects/meta_cl/experiments/anml_replicate.py:114]: Do we still need to leave it as 1, can we ramp it up to 5 again now we have InstanceNorm2d? Theoretically won't impact the results, and can make it faster.

File: projects/meta_cl/networks/anml_networks.py [hunk 2]
```diff
@@ -77,7 +77,7 @@ def conv_block(cls, in_channels, out_channels, kernel_size, stride, padding):
                 in_channels=in_channels, out_channels=out_channels,
                 kernel_size=kernel_size, stride=stride, padding=padding
             ),
-            nn.BatchNorm2d(num_features=out_channels),
+            nn.InstanceNorm2d(num_features=out_channels, affine=True),
```
Comment (lucasosouza) [inline projects/meta_cl/networks/anml_networks.py:80]: We can drop the train=True in the eval function, that is no longer required, and can bring some unexpected side effects in the future (for example if we include dropout in one of these networks). InstanceNorm2d has `truck_running_stats=False` as default.
